diff --git a/src/symjit/complexify.rs b/src/symjit/complexify.rs
--- a/src/symjit/complexify.rs
+++ b/src/symjit/complexify.rs
@@ -306,7 +306,7 @@ impl Generator for Complexifier {
 
     fn abs(&mut self, dst: Reg, s1: Reg) {
         if self.is_real_reg(s1) {
-            self.mir.abs(dst, s1);
+            self.mir.abs(re(dst), re(s1));
             self.set_reg_real(dst);
         } else {
             self.mir.times(Self::T0, re(s1), re(s1));
