From 4358f6f1bec771a22342f24ca71e8a7f56047c8d Mon Sep 17 00:00:00 2001
From: ValentinHirschi <valentin.hirschi@gmail.com>
Date: Sun, 26 Jul 2026 15:29:41 +0200
Subject: [PATCH 1/2] Build SymJIT as an rlib for Rust integration tests

---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 42221e3..34ef5c2 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -42,4 +42,4 @@ default = []
 [lib]
 # name = "_lib"
 path = "rust/lib.rs"
-crate-type = ["cdylib"]
+crate-type = ["cdylib", "rlib"]
